code/modules/cargo/blackmarket/blackmarket_items/cybernetics.dm

```diff
@@ -0,0 +1,268 @@
+
+
+// CYBERLINKS
+
+/datum/blackmarket_item/cyberlinks
+	category = "Cybernetics"
+	markets = list(/datum/blackmarket_market/cybernetics)
+
+/datum/blackmarket_item/cyberlinks/nt_low
+	name = "Nanotrasen's Basic Cyberlink Package"
+	desc = "We honestly have no clue why you are trying to purchase these off of us, but if you are willing to buy, we are willing to sell."
+	item = /obj/item/organ/cyberimp/cyberlink/nt_low
+	availability_prob = 60
+	stock_min = 2
+	stock_max = 5
+	price_min = 200
+	price_max = 400
+
+/datum/blackmarket_item/cyberlinks/nt_high
+	name = "Nanotrasen's Advanced Cyberlink Package"
+	desc = "More advanced form of NT cyberlink, allows for more implants, but may require some hacking to make older implants compatible with this."
+	item = /obj/item/organ/cyberimp/cyberlink/nt_high
+	availability_prob = 40
+	stock_min = 1
+	stock_max = 2
+	price_min = 300
+	price_max = 600
+
+/datum/blackmarket_item/cyberlinks/terragov
+	name = "Terragov Integrated Cybernetic Managment System"
+	desc = "Quite rare around this part of the galaxy, allows unrestricted connection to all of Terragov's implants, we don't guarantee it will work with any other cybernetic tough."
+	item = /obj/item/organ/cyberimp/cyberlink/terragov
+	availability_prob = 15
+	price_min = 700
+	price_max = 1000
+
+/datum/blackmarket_item/cyberlinks/syndicate
+	name = "Interdyne Cybernetic Operating Bioware"
+	desc = "A very rare cyberlink, used only by interdyne politicians, and other syndicate assault teams. Very rare find."
+	item = /obj/item/organ/cyberimp/cyberlink/syndicate
+	availability_prob = 5
+	price_min = 1000
+	price_max = 1200
+
+/// CYBERNETICS
+
+/datum/blackmarket_item/cybernetics
+	category = "Cybernetics"
+	markets = list(/datum/blackmarket_market/cybernetics)
+	stock = 1 //Can never have more than 1 of the same implant on the auction at the same time.
+	var/randomizable = TRUE
+
+/datum/blackmarket_item/cybernetics/spawn_item(loc)
+	. = ..()
+	var/obj/item/organ/cyberimp/implant = .
+	if(prob(30) && randomizable)
+		implant.random_encode()
+
+// LEGS
+
+/datum/blackmarket_item/cybernetics/table_glider
```

Comment:
these descriptions are so inconsistent, some refer to NT as NT and some refer to NT as Nanotrasen and I think I saw a NanoTrasen in there too. some of them end with a . at the end and some don't

Reply:
eh that's fine